apps/smtp/src/modules/smtp/services/mjml-compiler.ts

```diff
@@ -16,8 +18,7 @@ export class MjmlCompiler implements IMjmlCompiler {
     this.logger.debug("Trying to compile MJML template");
 
     const safeCompile = fromThrowable(
-      // Used require on purpose - when import where used, modules resolution failed
-      require("mjml"),
+      compile,
```

Comment:
https://github.com/saleor/apps/pull/1450/files#diff-8e0eeba230c77781145134423b9a18d060e954a1be1d2d04b7b08f70a8006424R19

for the context, please smoke test it